cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/goals/ResourceDistributionGoal.java

```diff
@@ -323,7 +330,7 @@ protected void rebalanceForBroker(Broker broker,
         LOG.debug("Successfully balanced {} for broker {} by moving out leaders.", resource(), broker.id());
         return;
       } else if (requireMoreLoad && !rebalanceByMovingLoadIn(broker, clusterModel, optimizedGoals,
-                                                             LEADERSHIP_MOVEMENT, optimizationOptions)) {
+                                                             LEADERSHIP_MOVEMENT, optimizationOptions, false)) {
```

Comment:
@efeg Hi, I'm curious why not use `moveImmigrantsOnly` but use `false` here?
It's merged pr. I am not sure you can notice this comment.